index.html

```diff
@@ -19,20 +19,22 @@
             <div class="nav-item" id="box3" ><a href="#"> Contact</a></div>
         </nav>
     </div>
-    </header>
-    <main>
+ 
+   
         <section>
             <div class="main-container">
 
                 <div id="main-item1">
                     <h2>Hey there. <span id="main-span1">I'm Kenny.</span> <span id="main-span2">I'm a software developer</span></h2>
                 </div>
                 <div id="main-item2" class="font-normal">
+                    <div id="main-p">
                     <p>
                     I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                     don’t hesitate to contact me.
                     </p>
                 </div>
+                </div>
```

Comment:
- [ ] Kindly 😃 as stated by the previous reviewer, If you check the Figma template, you'll see that there's a button with a down arrow under the social media icons,  so please I suggest adding it to stick as much as possible to the design.💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
| ![image](https://github.com/Armynerh/personal_portfolio/assets/83360636/37d4f2a1-5efb-4600-b3c6-a2e860e981e8)    |     ![image](https://github.com/Armynerh/personal_portfolio/assets/83360636/a5adb9a9-cc32-4092-a981-600350d30b4c)  |